server/routes/auth.js

```diff
@@ -55,15 +56,15 @@ router.post('/signup', (req, res) => {
             } else {
                 res.status(500).send({ message: 'Error creating user' })
             }
-        })
+        });
     })
     .catch(err => {
         console.log('Error in POST /auth/signup', err)
         res.status(503).send({ message: 'Database or server error' })
-    })
```

Comment:
Somewhere in this file there is a mispelling of validPassword I think, I think it has an extra i, I ran into it when debugging usersignup. If you could double check that it's a thing, and that it's spelled correctly I would appreciate it.

Reply:
THE HEATHEN HAS BEEN FOUND AND PURGED.